src/btree/row_modify.c

```diff
@@ -346,33 +342,36 @@ __wt_update_obsolete_check(
      * Walk the list of updates, looking for obsolete updates at the end.
      *
      * Only updates with globally visible, self-contained data can terminate update chains.
-     *
      */
     for (first = NULL, count = 0; upd != NULL; upd = upd->next, count++) {
         if (upd->txnid == WT_TXN_ABORTED)
             continue;
 
-        ++upd_seen;
+        /*
```

Comment:
Keep only this change and revert the other changes in this backport because of use of those variables.